example/map/terra_test.c

```diff
@@ -1,5 +1,7 @@
-#include <unistd.h>
 #include <zeo/zeo_map.h>
+#ifndef __WINDOWS__
+#include <unistd.h>
+#endif
```

Comment:
Please include unistd.h before any other header files. Namely,
```C
#ifndef __WINDOWS__
#include <unistd.h>
#endif
#include <zeo/zeo_map.h>
```
is correct.
Please also check if unlink is available on MSC.

Reply:
The following is more preferable.
```C
#ifdef __unix__
#include <unistd.h>
#endif
#include <zeo/zeo_map.h>
```